ecma402_traits/src/numberformat.rs

```diff
@@ -38,7 +38,13 @@ pub mod options {
     /// The ISO currency code for a currency, if the currency formatting is being used.
     /// For example "EUR" or "USD".
     #[derive(Eq, PartialEq, Debug, Clone)]
-    pub struct Currency(&'static str);
+    pub struct Currency(pub String);
+
+    impl From<&str> for Currency {
```

Comment:
Consider adding a comment about the lack of validation.

Reply:
Done.